packages/client/public/imtr/transformed/bmf5XsTrZpgTkbJoP.json

```diff
@@ -1,80 +1,80 @@
 {
   "permits": [
     {
-      "version": 1,
+      "version": 2,
```

Comment:
Had to trigger some new versions to be able to download imtr files again. This was because of a release by FloLegal.